WordPressVIPMinimum/Sniffs/VIP/RestrictedFunctionsSniff.php

```diff
@@ -27,7 +30,17 @@ public function getGroups() {
 				'message' => '%s is prohibited on the WordPress.com VIP platform',
 				'functions' => array( 'get_super_admins' ),
 			),
+			'get_children' => array(
+				'type' => 'error',
+				'message' => '%s() performs a no-LIMIT query by default. Make sure to set a reasonable posts_per_page, get_children will do a -1 query by default, a maximum of 100 should be used.',
```

Comment:
I think the punctuation on this message might be a bit ambiguous.

I suggest something like:

`%s() performs a no-LIMIT query by default, make sure to set a reasonable posts_per_page.  get_children will do a -1 query by default, a maximum of 100 should be used.`